jasmine.js

```diff
@@ -0,0 +1,39 @@
+describe("array numbers", function() {
+
+	it("check elements from array", function() {
+	 	expect(tests.firstTest()).toEqual(jasmine.any(Number));
```

Comment:
I expect the test  to verify that the function returns the `correct number` not `any number`